Article/Article.js

```diff
@@ -3,21 +3,23 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton')
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand'
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle())
```

Comment:
Well done to use an arrow function as your callback function on this event lister, as it is now implicitly binding the 'this' keyword so that your expandArticle method works correctly.